src/main/java/mods/railcraft/common/blocks/machine/beta/TileEngineSteamHobby.java

```diff
@@ -218,6 +217,33 @@ public boolean canExtractItem(int index, ItemStack stack, EnumFacing direction)
     }
 
     @Override
+    public ItemStack removeStackFromSlot(int index) {
+        ItemStack stack = getStackInSlot(index);
```

Comment:
Both of those were copy/pasted, a side effect of suddenly having even more methods to support.